xbmc/pvr/recordings/PVRRecording.h

```diff
@@ -69,6 +69,7 @@ namespace PVR
     std::string   m_strThumbnailPath; /*!< thumbnail path */
     std::string   m_strFanartPath;    /*!< fanart path */
     unsigned      m_iRecordingId;     /*!< id that won't change while xbmc is running */
+    std::string   m_strEpisodeName;   /*!< name of the episode */
```

Comment:
Please remove the showtitle variable in favor to the episodename variable. The former is not needed anymore.

Reply:
I think m_strShowTitle is defined in another file, in VideoInfoTag.h. But I am not sure if it makes sense to replace it everwhere.
In RecentlyAddedJob.cpp line 103/4 currently m_strShowTitle is assigned to ShowTitle and m_strTitle is assigned to EpisodeTitle. Replacing m_strShowTitle here seems to be wrong.

Or is m_strShowTitle independent from m_strShowTitle in recordings?

Reply:
Oh well, my fault. `m_strShowTitle` is defined in `CVideoInfoTag`from which `CPVRRecording`is derived. I overlooked this.

More interesting: CVideoInfoTag already exposes: 
* `m_strShowTitle` as property `showtitle`. 
* `m_iSeason`as property `season`
* `m_iEpisode` as property `episode`

As above `CVideoInfoTag`members get filled by `CPVRRecording`, every pvr recording already should expose the episode name, episode number and season number via its base class. If this is working as designed, there is no need to extend `CPVRRecording` which would render this PR "useless" as it adds no new functionality.